src/sentry/api/endpoints/project_stacktrace_link.py

```diff
@@ -237,20 +258,12 @@ def get(self, request: Request, project: Project) -> Response:
                     break
 
             # Post-processing before exiting scope context
-            found: bool = result["sourceUrl"] is not None
```

Comment:
Moving all of these removed lines within `update_tags`

Reply:
I wish github had a nicer way to view moved code blocks - thanks for calling it out!

Reply:
I wish it to! I have a hacky trick were I place the old code and new code side by side in two different windows (You can hide/show comments from a UI drop down)

<img width="1033" alt="image" src="https://user-images.githubusercontent.com/44410/211086036-3e3499f7-135e-4906-9cb7-a3563ae682fa.png">
<img width="169" alt="image" src="https://user-images.githubusercontent.com/44410/211086099-a09d9d24-1436-4d4f-ae92-828d3e4bf54c.png">